lesson_00/anthonymays/Nile Jackson's README

```diff
@@ -0,0 +1,37 @@
+$ git clone
+
+Nile Jackson's README.md
+
+Introduction
+
+Hi, my name is Nile Jackson. I started learning javascript about a month ago, and I'm about two weeks into my journey with java; I'm new. I decided to learn programming because of my fascination with digital audio workstation plugins, I plan on building my own in due time. Before code differently, I worked primarily in juvenile detention; this wasn't fulfilling so a little over a year ago I rented a room in brooklyn New York and began an audio engineering internship at a place called Jungle studios. After two months at jungle and several odd jobs I landed my first "gig" as the sound guy at a local bar called skytown. I continued working small gigs for a few months and traveling back to delaware 3 times a week to do carpentry work with a long time friend of mine. Eventually I decided that this life style wasn't sustainable, so after another four month internship at Manhattan Beach studios, I did some research and discovered code differently.
+
+Likes
+
+I enjoy cooking because I'm an eight year vegan who misses the dishes I grew up on, namely jamaican food; I spend a lot of down time in the kitchen trying to "veganize" things. Other than food, I love to read, I hated books as a kid but four/five years ago I moved to Iowa to play(school) football at a small private college called William Penn, I wasn't a fan of Iowa or the school so I began using the library as a place to escape. I also like my two dogs, Pico(a pitbull/great dane mix) & LuLu(a small mut of some sort).
+
+Dislikes
+
+I'm not a fan of the following; artichokes, english breakfast tea, the new york subway system, and excessively cold winters.
```

Comment:
Make an actual list using markdown syntax.

Reply:
Okay, I made an actual list with the asterisks to correct the syntax